### DISCUSSION TITLE
Turn on `print_stdout` clippy lint

### DISCUSSION THREAD

**Discussion by dsherret** [swc-project/swc#3245] [Ideas]
Given that swc has a lot more contributors, I think it would be useful to slowly start turning on some clippy rules. For example, the "print_stdout" rule would be useful on most crates:

https://rust-lang.github.io/rust-clippy/master/#print_stdout

This would prevent https://github.com/swc-project/swc/pull/3244

Edit: Actualy this lint rule is annoying to use until this is fixed https://github.com/rust-lang/rust-clippy/issues/6610

**Comment by kdy1** [comment]
I think using clippy is a good thing to do. Recent nightly typically has all components so my reason for not using clippy is not valid anymore.
I'll configure it soon.

**Reply by kdy1** [reply]
I tried it, but it prodecd too much warnings so it will take lots of time.
![image](https://user-images.githubusercontent.com/29931815/149304629-23321ac3-41c1-4799-b44c-0b0e005df918.png)

**Reply by kdy1** [reply]
Oh, I'll see if `--fix` works
